transformations/diverse_paraphrase/requirements.txt

```diff
@@ -0,0 +1,3 @@
+# for diverse paraphrase dips
+nltk==3.6.2
+torchtext
```

Comment:
Is there any particular reason for not adding version for torchtext?

Reply:
The main `requirements.txt` file does not have a specific version for `torch`.
`torchtext` depends on the original torch version. 

If the version for the original torch is added, this can be fixed too.

Reply:
@kaustubhdhole What's your opinion? Shall we specify a torch version in the main requirement file?

Reply:
I avoided specifying the torch version to make it quicker for colab users - Colab has a prebuilt torch installed. (Torch takes a lot of time to download) Its fine to keep torchtext that way. A good idea would be to mention in the README which versions of torchtext are preferred (if there's any such preference).

Reply:
There is no preference. 

I just tested the code with torch==1.9.0 and torchtext==0.10.0. Although the useful functions in torchtext are present in all the earlier available versions too, and is unlikely to be removed in the future.